AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/vh-participant.ts

```diff
@@ -0,0 +1,132 @@
+import { InterpreterSelectedDto } from 'src/app/booking/interpreter-form/interpreter-selected.model';
+import { ScreeningDto } from 'src/app/booking/screening/screening.model';
+import { LinkedParticipantModel } from './linked-participant.model';
+import { HearingRoleCodes, HearingRoles } from './hearing-roles.model';
+import { v4 as uuid } from 'uuid';
+import { cloneWithGetters } from '../helpers/clone-with-getters';
+
+export class VHParticipant {
+    id?: string;
+    externalReferenceId?: string;
+    title?: string;
+    firstName?: string;
+    lastName?: string;
+    middleNames?: string;
+    displayName?: string;
+    username?: string;
+    email?: string;
+    hearingRoleName?: string;
+    hearingRoleCode?: string;
+    phone?: string;
+    representee?: string;
+    company?: string;
+    isExistPerson?: boolean;
+    interpreterFor?: string;
+    linkedParticipants?: LinkedParticipantModel[];
+    interpreteeName?: string;
+    isInterpretee?: boolean;
+    userRoleName?: string;
+    isCourtroomAccount?: boolean;
+    addedDuringHearing?: boolean;
+    contactEmail?: string;
+    isJudiciaryMember?: boolean;
+    interpretation_language: InterpreterSelectedDto;
+    screening?: ScreeningDto;
+    // flag to indicate if participant is the last in the list and don't need decoration bottom line
+    flag?: boolean;
```

Comment:
angular ngFor loops have this built-in using the `isLast` field. If it's just for display purposes better to leave it with ng instead of us managing it
```
<div *ngFor="let item of items; let isLast=last">
  {{ item }}
  <ng-container *ngIf="isLast">This is the last item!</ng-container>
</div>
```

Reply:
thanks, this was from the original view model that was migrated across. There might be other properties like this that were just added for convenience

Reply:
have removed this property